### PR TITLE
bpo-39273: Expose BUTTON5_* constants in the curses module if available

### PR DESCRIPTION
**Pull request by ZackerySpytz**
https://bugs.python.org/issue39273

### CODE REVIEW

File: Modules/_cursesmodule.c [hunk 1]
```diff
@@ -4691,6 +4691,14 @@ PyInit__curses(void)
     SetDictInt("BUTTON4_DOUBLE_CLICKED",   BUTTON4_DOUBLE_CLICKED);
     SetDictInt("BUTTON4_TRIPLE_CLICKED",   BUTTON4_TRIPLE_CLICKED);
 
+#if NCURSES_MOUSE_VERSION > 1
```
Comment (serhiy-storchaka) [inline Modules/_cursesmodule.c:4873]: `NCURSES_MOUSE_VERSION` is a ncurses specific constant. It is not defined on other implementations. Add a check that it is defined.

**Review by lisroach — APPROVED** [review]
LGTM

**Review by serhiy-storchaka — CHANGES_REQUESTED** [review]

**Review by serhiy-storchaka — APPROVED** [review]

**Review by serhiy-storchaka — COMMENTED** [review]
Please update the documentation.

**Review by serhiy-storchaka — CHANGES_REQUESTED** [review]
Please update `Doc/library/curses.rst` and `Doc/whatsnew/3.10.rst`.

**Comment by ZackerySpytz** [comment]
I have made the requested changes; please review again.

**Review by serhiy-storchaka — APPROVED** [review]
:+1:

**Pull request merged**